README.md

```diff
@@ -1,2 +1,36 @@
 # scs-pki-research
 PKI research material
+
+# Example signing:
+
+Generate SHA256 hash
+
+`$ sha256sum 0r434a8n04i14s4sp3fam23cplwyn1iw-generic-x86_64-debug-nixos.img 
+1e648d85a8ca1c6f55fcc060bbcbd24a4ff2deb5844e198088689ca840f3337a  0r434a8n04i14s4sp3fam23cplwyn1iw-generic-x86_64-debug-nixos.img`
+
+Sign the hash with Yubikey
+
+`$ ./start.sh --sign --h=1e648d85a8ca1c6f55fcc060bbcbd24a4ff2deb5844e198088689ca840f3337a
+YubiKey found on USB Bus: 003, Device: 002
+499OENQ5B08XVLL4nBJdeoWfYl2TbQEatp1SxeDMnqf527ie6OVBrNCh9SmfCD24rw6v/C/QQ13OOBHBehKQEA==`
+
+Verify the signature (Valid case)
```

Comment:
1. Verify the signature (valid case):
2. Add an empty line before and after the code line.